src/pages/dashboard/_TotalAndPositiveLegend.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import clsx from 'clsx'
+
+import { formatNumber, formatDate } from '../../utilities/visualization'
+import styles from './total-and-positive-legend.module.scss'
+
+export default ({ date = null, positive, total, perCapita = false }) => (
+  <>
+    {date && (
+      <span className={clsx(styles.item, styles.date)}>{formatDate(date)}</span>
+    )}
+    <div className={styles.item}>
+      <span>{formatNumber(total)}</span>{' '}
+      <span className={clsx(styles.legendText, styles.total)}>
+        tests {perCapita && 'per million'}
```

Comment:
I'm not sure if this is clear enough. Per million _what_? I'm tempted for it to be "tests per million people", but recognize that will be a lot of text to fit in the tooltip. Maybe smaller font size would make it possible...

Alternatively, maybe we footnote the "Tests per capita" text in the toggle, like we're doing with the "Stay at home" legend. So "per capita" is explained overall as a later aside. Then the labels in the tooltip could be "tests per capita" and "positive tests per capita". cc @julialedur

Reply:
@vijaysharwar yeah, I was thinking the same thing! 
I think the best solution is to do "tests per capita" on the tooltip
I think the note should be at the top though, cause it's an important information to know before you look at the graphic. 

so my suggestion is to either change the toggle button to say "Tests Per One Million People" (like the screenshot below) or to write "Per capita = per one million people" right below the toggle. I think the second option solves the long button issue, but is a bit repetitive

<img width="371" alt="Screen Shot 2020-04-18 at 5 25 07 PM" src="https://user-images.githubusercontent.com/39416238/79671624-d9aa0880-8199-11ea-875b-798d6f1dcab1.png">

Reply:
At risk of nitpicking... I think I prefer keeping the toggle as is. It works best visually if the two sides are as equal width as possible. Plus, changing the width of the toggle means ensuring the layout doesn't break across viewports, especially small/medium screens. Don't forget that @nselikoff added "Display by:" text in front of the toggle as well, via https://github.com/COVID19Tracking/website/pull/627#discussion_r410715402. So we're tight for horizontal space at the point that the legend goes onto same line as the toggle.

Elsewhere, in the map section, we're doing it this way:

> Per capita*
> 
> \* Per one million people

Might we be consistent? Use "Tests Per Capita*" text in both the toggle and tooltips. Then put a note below with the other footnote. It's findable if people _need it_, but otherwise doesn't interfere with usability and layout. Like so:

<img width="1115" alt="Screen Shot 2020-04-18 at 7 00 50 PM" src="https://user-images.githubusercontent.com/63169602/79673408-71faba00-81a7-11ea-9a4f-a3e28b0c5593.png">

Reply:
yes! that's great!